dont_have_vinogret.py

```diff
@@ -0,0 +1,28 @@
+
+import datetime
+import random
+
+
+def random_date_in_range_str(start, end):
+    """
+    The function choose random date in the range between the 2 given dates
```

Comment:
The function chooses* a* random date

Reply:
I didnt understad this not, what is the problem?

Reply:
It's a small grammer mistake - you should write "The function chooses a random date" instead of what's written.